Chat space チャット画面(View)

T-kumaX opened the pull request:
#what
チャット画面の作成(ビュー)

#why
ビューから作成し、後から細かい機能をつけていくため

T-kumaX commented:
<img width="1440" alt="2018-11-21 6 58 38 1" src="https://user-images.githubusercontent.com/44967219/48805680-52195980-ed5b-11e8-9acb-3ba2b8efc23e.png">

hirozak commented on chat-space/app/assets/stylesheets/application.scss:
```diff
@@ -0,0 +1,192 @@
+html, body {
```
このファイルはimportのみ行ってください

hirozak commented on chat-space/app/assets/stylesheets/application.scss:
```diff
@@ -0,0 +1,192 @@
+$Iron_blue: #434A54;
+$Dark_brue: #2f3e51;
+$Light_brue: #668ad8;
+$White: white;
```
この記述はvariableで行うのではないでしょうか

hirozak commented on chat-space/app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,31 @@
+      .header-left
+        %h1.header-left--user_name seo
+        %i.fa.fa-cog.fa-lg
+        %i.fa.fa-edit.fa-lg
```
ヘルパーメソッド使いましょう

hirozak commented on chat-space/app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,31 @@
+    %link{:href => "style.css", :rel => "stylesheet"}/
+  %body
+    %header.header-right
+      %input{:style => "color: #5D99FF;", :type => "button", :value => "Edit"}/
```
ハッシュロケットでなくシンボルを使いましょう

hirozak commented on chat-space/app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,31 @@
+      %h1.contents--user1 seo
+      %p.contents--time1 2018/11/12 11:00:20
+      %p.contents--user_comment1 Hellow world!
+      %h1.contents--user2 neko
```
1,2などと数字をつける必要はありません

T-kumaX commented:
%button_tag fa_icon("cog").fa-log 
%button_tag fa_icon("edit").fa-log 
ヘルパーメソッドにするのはこの形で合ってますか？

Commit:
Add root 'messages#index'  To  routes.rb

Commit:
Add  チャット画面  To  messages/index.html.haml

Commit:
Add  application.html.haml

headの設定

Commit:
Delete  application.html.erb

Commit:
Add index action  To  messages_controller.rb

Commit:
Add @import (font-awesome, reset, style)  To  application.scss

Commit:
Delete application.css

Commit:
Add top-page scss  To  _style.scss

Commit:
Add _reset.scss

T-kumaX commented:
#what
device機能の追加

#why
ログイン、新規登録をできるようにするため

Commit:
Add  gem 'devise'  To  Gemfile

Commit:
Add devise gem To  Gemfile.lock

Commit:
Execution  $ rails g devise:install  Add  config/initializers/devise.rb

Commit:
Execution  $ rails g devise:install  To  config/locales/devise.en.yml

Commit:
Execution  migrate  To  20190104030808_devise_create_users.rb

t.string :name, null: false, unique: true
の追記

Commit:
Add schema.rb

20190104030808 DeviseCreateUsers: migrated

Commit:
Execution  $ rails g devise user  To  models/user.rb

Commit:
Execution  $ rails g devise:views   Add  devise file

registrations/new.html.haml、sessions/new.html.haml、shared/_links.html.ha
ml は編集済み

Commit:
Add _user.scss

Commit:
Edit  application_controller.rb  about devise

Commit:
Add  users_controller.rb

edit, updateアクション　
ストロングパラメーター  user_paramsアクション

Commit:
Add  messages/_side_ber.html.haml

仮置きの名前を= current_user.name に変更

Commit:
Add users/edit.html.haml

Commit:
Add _notifications.html.haml

フラッシュメッセージの実装

Commit:
Delete  'data-turbolinks-track': 'reload'   Add  = render 'layouts/notifications'   To  application.html.haml

フラッシュメッセージの実装

Commit:
Add _variable.scss

フラッシュメッセージの色を指定

Commit:
Add  _flash.scss

フラッシュメッセージ scss

Commit:
Add  @import  To  application.scss

user, variable, flash

Commit:
Edit application.rb  Add  config.i18n.default_locale = :ja

フラッシュメッセージの日本語化

Commit:
Add devise.ja.yml

フラッシュメッセージの日本語化

Commit:
Execution  $ rake haml:replace_erbs  To  layouts/mailer view files

Delete erb file
Add haml file
